lightning/src/onion_message/dns_resolution.rs

```diff
@@ -211,16 +211,21 @@ pub struct HumanReadableName {
 impl HumanReadableName {
 	/// Constructs a new [`HumanReadableName`] from the `user` and `domain` parts. See the
 	/// struct-level documentation for more on the requirements on each.
```

Comment:
Can you mention the new filtering in the struct docs?

Reply:
Done:
```rust
/// The `user` and `domain` parts combined cannot exceed 231 bytes in length;
/// each DNS label within them must be non-empty and no longer than 63 bytes.
```